data/reusables/gated-features/code-scanning.md

```diff
@@ -1,6 +1,4 @@
 {% ifversion fpt or ghec %}{% data variables.product.prodname_code_scanning_capc %} is available for all public repositories, and for private repositories owned by organizations where {% data variables.product.prodname_GH_advanced_security %} is enabled.
 {%- elsif ghes > 3.0 %}{% data variables.product.prodname_code_scanning_capc %} is available for organization-owned repositories where {% data variables.product.prodname_GH_advanced_security %} is enabled.
-{%- elsif ghae %}
-{% data variables.product.prodname_code_scanning_capc %} is available as part of {% data variables.product.prodname_GH_advanced_security %}, which is free during the beta release.
-{%- else %}
-{% data variables.product.prodname_code_scanning_capc %} is available if you have a license for {% data variables.product.prodname_GH_advanced_security %}.{% endif %} {% data reusables.advanced-security.more-info-ghas %}
+{%- elsif ghes = 3.0 %}
+{% data variables.product.prodname_code_scanning_capc %} is available if you have a license for {% data variables.product.prodname_GH_advanced_security %}.{% endif %}{% ifversion not ghae %} {% data reusables.advanced-security.more-info-ghas %}{% endif %}
```

Comment:
I appreciate Dorothy's concerns about maintaining information across multiple pages, but I suspect she wasn't aware that this was a single reusable.

I agree with @mchammer01 that we should modify rather than delete the text for GHAE users. This ensures that they're aware that this part of GHAS and there will be charges involved. It also ensures that they have a link to the article about GHAS if they have any questions about what this means.

```suggestion
{%- elsif ghae %}
{% data variables.product.prodname_code_scanning_capc %} is available as part of {% data variables.product.prodname_GH_advanced_security %}.
{%- elsif ghes = 3.0 %}
{% data variables.product.prodname_code_scanning_capc %} is available if you have a license for {% data variables.product.prodname_GH_advanced_security %}.{% endif %}{% ifversion not ghae %} {% data reusables.advanced-security.more-info-ghas %}{% endif %}
```